airbyte-integrations/bases/base-normalization/dbt-project-template/macros/cross_db_utils/type_conversions.sql

```diff
@@ -38,3 +38,8 @@
 {% macro redshift__cast_to_boolean(field) -%}
     cast(decode({{ field }}, 'true', '1', 'false', '0')::integer as boolean)
 {%- endmacro %}
+
+{# cast_to_bigint -------------------------------------------------     #}
+{% macro mysql__type_bigint() %}
```

Comment:
Should this go into `airbyte-integrations/bases/base-normalization/dbt-project-template/macros/cross_db_utils/datatypes.sql` instead?

this is extending the macro from dbt_utils right?